### PR TITLE
CRM-19947 fix file link on custom multi profile edit form

### PR DESCRIPTION
**Pull request by lcdservices**
* [CRM-19947: custom multi profile: file upload display issue](https://issues.civicrm.org/jira/browse/CRM-19947)

### CODE REVIEW

File: CRM/Profile/Form.php [hunk 1]
```diff
@@ -500,7 +500,7 @@ public function setDefaultsValues() {
             ) {
               $entityId = $this->_activityId;
             }
-            $url = CRM_Core_BAO_CustomField::getFileURL($entityId, $key);
+            $url = CRM_Core_BAO_CustomField::getFileURL($entityId, $key, $value);
```
Comment (jitendrapurohit) [inline CRM/Profile/Form.php]: I was able to replicate the issue even after this patch. Maybe, even if `$value` is passed as NULL, `getFileURL()` returns the first file attached to the entity if multiRecordWhereClause is not being passed to the function. 

As we are setting defaults here, I think we should check for $value being set before calling to return the url. Something like:

    $url = '';
    if (isset($value)) {
      $url = CRM_Core_BAO_CustomField::getFileURL($entityId, $key, $value);
    }

It works for me. Can you please try this and check if it works for you as well ?

Comment (lcdservices) [inline CRM/Profile/Form.php]: yes, I think that works fine.

Comment (jitendrapurohit) [inline CRM/Profile/Form.php]: @lcdservices Can you please update this PR so that we can merge and close the ticket?

Comment (lcdservices) [inline CRM/Profile/Form.php]: done. PR has been updated.

**Review by jitendrapurohit — APPROVED** [review]

**Pull request merged**